skycultures/Book_of_Fixed_Stars_English/asterism_names.eng.fab

```diff
@@ -0,0 +1,185 @@
+﻿01UMi01	_("The two oryx calves")

```

Comment:
That's a really interesting addition! :-)

Reply:
Asterisms fits nicely as a way to display Al-Sufi mapping of indigenous Arab star names to Ptolemaic constellations.